packages/main/test/pages/Carousel.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+
+	<title>Carousel</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); padding: 1rem;">
+<style>
+	ui5-carousel {
+		height: 20rem;
+		margin-bottom: 1rem;
+	}
+</style>
+<ui5-carousel id="carousel1" cycling="true">
+	<ui5-button>Button 1</ui5-button>
+	<ui5-button>Button 2</ui5-button>
+	<ui5-button>Button 3</ui5-button>
+</ui5-carousel>
+
+<ui5-carousel id="carousel2">
```

Comment:
If you set items-per-page="4" or "2" (or a number that is not divider of 9) to the carousel for example with id=carousel2, there is a JS error in the console and the carousel stops working.

If we want to support these cases:
In case of  items-per-page="4"- the split should be 4, 4, 1
In case of  items-per-page="2" - 2, 2, 2, 2, 1
(items-per-page="3" is working fine)